ister.py

```diff
@@ -408,7 +413,7 @@ def copy_os(args, template, target_dir):
         swupd_env["http_proxy"] = template["HTTPProxy"]
         LOG.debug("http_proxy: {}".format(template["HTTPProxy"]))
 
-    run_command(swupd_command, environ=swupd_env)
```

Comment:
I'd rather avoid having to add an extra option that is essentially display swupd output... So I think this default is prefered.